decide/src/test/java/group17/LicAnalyzerTest.java

```diff
@@ -426,10 +426,148 @@ public void lic7InvalidK_PTS() {
 
 ///////////////////////////// Lic 8 /////////////////////////////
 
-    @Test
-    public void lic8Test() {
-        assertTrue(true);
-    }
+@Test
+public void lic8TestUncontained() {
+    input.NUMPOINTS = 5;
+    input.A_PTS = 1;
+    input.B_PTS = 1;
+    input.X_COORD = new double[]{-2.0, 13.37, 0.0, 13.37, 2.0};
+    input.Y_COORD = new double[]{0.0, 13.37, 0.0, 13.37, 0.0};
+    input.RADIUS1 = 1.5;
+
+    assertTrue(licAnalyzer.lic8(input));
+}
+
+@Test
+public void lic8TestUncontainedOffset1() {
+    input.NUMPOINTS = 6;
+    input.A_PTS = 1;
+    input.B_PTS = 1;
+    input.X_COORD = new double[]{13.37, -2.0, 13.37, 0.0, 13.37, 2.0};
+    input.Y_COORD = new double[]{13.37, 0.0, 13.37, 0.0, 13.37, 0.0};
+    input.RADIUS1 = 1.5;
+
+    assertTrue(licAnalyzer.lic8(input));
+}
+
+public void lic8TestContained() {
+    input.NUMPOINTS = 6;
+    input.A_PTS = 1;
+    input.B_PTS = 1;
+    input.X_COORD = new double[]{13.37, -2.0, 13.37, 0.0, 13.37, 2.0};
+    input.Y_COORD = new double[]{13.37, 0.0, 13.37, 0.0, 13.37, 0.0};
+    input.RADIUS1 = 3;
+
+    assertFalse(licAnalyzer.lic8(input));
+}
+
+public void lic8TestContainedOffset1() {
+    input.NUMPOINTS = 6;
+    input.A_PTS = 1;
+    input.B_PTS = 1;
+    input.X_COORD = new double[]{13.37, -2.0, 13.37, 0.0, 13.37, 2.0};
+    input.Y_COORD = new double[]{13.37, 0.0, 13.37, 0.0, 13.37, 0.0};
+    input.RADIUS1 = 3;
+
+    assertFalse(licAnalyzer.lic8(input));
+}
+
+@Test
+public void lic8TestContainedBoundary() {
+    input.NUMPOINTS = 5;
+    input.A_PTS = 1;
+    input.B_PTS = 1;
+    input.X_COORD = new double[]{-2.0, 13.37, 0.0, 13.37, 2.0};
+    input.Y_COORD = new double[]{0.0, 13.37, 2.0, 13.37, 0.0};
+    input.RADIUS1 = 2;
+
+    assertFalse(licAnalyzer.lic8(input));
+}
```

Comment:
Asking just to make sure, this test asserts correct consistently? Asking since when I looked inside the method `minimumEnclosingRadius` in `GeometryUtils.java` I can see that it uses `==` on floating points. 

This could be an issue related to something else since I currently having some difficulties understanding how `minimumEnclosingRadius` works/what formula it uses so just want to make sure.

Reply:
Haven't tested with other tests, but if GeometryUtils.java would be changed to support more accurate float comparison then this lic would also benefit from that improvement.